pkgs/record_use/lib/record_use_internal.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+export 'src/data_classes/annotation.dart';
```

Comment:
It's really hard to see what's different between `lib/record_use.dart` and `lib/record_use_internal.dart`. Why not put internal things into their own library?

Reply:
You mean in their own packages? Or just change the imports/exports?

Reply:
I mean all the libraries/files in `src/data_classes/*` seem to have public and private things in them (so you `show X` specific entries).

Wouldn't it be easier to have libraries containing only the public API elements and other libraries that contain private things?

Reply:
I think they don't really - I just like `show` keywords personally. The only thing not exported is the `Reference` superclass, I now added a comment in the exports about this.